### PR TITLE
Issue #SC-1090 feat: adding es module into user org service

### PR DESCRIPTION
**Pull request by manzarul**
adding new module for elasticsearch inside user-org repo, this module is capable for handling both es tcp client call as well as rest client call.

### CODE REVIEW

File: user-org-es-util/src/main/java/org/sunbird/es/service/impl/ElasticSearchRestHighImpl.java [hunk 1]
```diff
@@ -0,0 +1,652 @@
...
+    SearchRequest searchRequest = new SearchRequest(index);
+    searchRequest.types(_DOC);
+
+    // check mode and set constraints
+    Map<String, Float> constraintsMap = ElasticSearchHelper.getConstraints(searchDTO);
+
+    BoolQueryBuilder query = new BoolQueryBuilder();
+
+    // add channel field as mandatory
+    String channel = PropertiesCache.getInstance().readProperty(EsConstant.SUNBIRD_ES_CHANNEL);
```
Comment (amitec006) [inline user-org-es-util/src/main/java/org/sunbird/es/service/impl/ElasticSearchRestHighImpl.java]: Are we using SUNBIRD_ES_CHANNEL? Is this check required

File: user-org-es-util/src/main/java/org/sunbird/es/service/impl/ElasticSearchRestHighImpl.java [hunk 2]
```diff
@@ -0,0 +1,652 @@
...
+    if (!(StringUtils.isBlank(channel) || EsConstant.SUNBIRD_ES_CHANNEL.equals(channel))) {
```
Comment (amitec006) [inline user-org-es-util/src/main/java/org/sunbird/es/service/impl/ElasticSearchRestHighImpl.java]: If we are using SUNBIRD_ES_CHANNEL, then all env variable should be at one place(externalResource properties file)

File: user-org-es-util/src/main/java/org/sunbird/es/service/impl/ElasticSearchRestHighImpl.java (hunk 1 above)
Comment (manzarul) [reply user-org-es-util/src/main/java/org/sunbird/es/service/impl/ElasticSearchRestHighImpl.java]: I don't know this is required or not, On Code threre was as comment as "adding channel field as mandatory" ,  i ask with devops team for that env if it's set then we have to use it other wise will discuss and remove this.

Comment (manzarul) [reply user-org-es-util/src/main/java/org/sunbird/es/service/impl/ElasticSearchRestHighImpl.java]: After Discussion with devops we found this env is not in used , so we removed it.

File: user-org-es-util/src/main/java/org/sunbird/es/service/impl/ElasticSearchRestHighImpl.java (hunk 2 above)
Comment (manzarul) [reply user-org-es-util/src/main/java/org/sunbird/es/service/impl/ElasticSearchRestHighImpl.java]: This is removed now.

**Review by indrajra — APPROVED** [review]

**Pull request merged**